README.md

```diff
@@ -27,6 +27,7 @@ Deploy and log in with username *admin* and the password that's set up during de
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.2</b> or higher
  * [Composer](https://getcomposer.org/)
+ * NodeJS <b>8.0</b> or higher
```

Comment:
How did you come to the conclusion 8.0 was the minimum version?

Reply:
Truthfully I haven't done extensive testing. 

v10 is the latest LTS though and v12 is current so I think v8 is an appropriate ask.